crates/ursa-network/src/measurements/mod.rs

```diff
@@ -69,29 +75,37 @@ impl Default for MeasurementManager {
 pub struct Measurements {
     pub bandwidth: Option<BytesPerSecond>,
     pub latency: Option<Milliseconds>,
+    pub uptime: Option<Milliseconds>,
 }
 
 struct PeerMeasurementManager {
     bandwidth: Bandwidth,
     latency: Latency,
+    uptime: Uptime,
 }
 
 impl PeerMeasurementManager {
     fn new() -> Self {
         Self {
             bandwidth: Bandwidth::new(),
             latency: Latency::new(),
+            uptime: Uptime::new(),
         }
     }
 
     #[allow(dead_code)]
     fn get_measurements(&self) -> Option<Measurements> {
```

Comment:
No, this function will be called to collect all of the measurements before sending them to the validators. The code for sending the measurements to the validators isn't there yet, so I just added this to prevent warnings.